cmd/ebitenmobile/gomobile.go

```diff
@@ -212,6 +218,16 @@ import (
 	return tmp, nil
 }
 
+func toolchainParameter() (string, error) {
+	pattern := regexp.MustCompile(`\bgo\d+\.\d+(\.\d+)?`)
+	rawVersion := runtime.Version()
+	results := pattern.FindStringSubmatch(rawVersion)
+	if len(results ) < 1 {
+		return "", errors.New("runtime.Version() regex did not capture any results!")
+	}
```

Comment:
```go
if len(results) == 0 {
    return "", fmt.Errorf("ebitenmobile: unexpected version: %s", rawVersion)
}
```

Also, please do not forget to run gofmt.

Reply:
Sorry, I hastily ran go fmt and missed the other part of your comment. This is addressed with latest push.